inst/paramtest/test_paramtest_mgcv_classif_gam.R

```diff
@@ -0,0 +1,57 @@
+library(mlr3extralearners)
```

Comment:
Would you mind just trying out removing this and seeing if the test still passes?

```suggestion
```